TODO Api/app/routes/router.py

```diff
@@ -0,0 +1,99 @@
+from fastapi import FastAPI, HTTPException, Depends, APIRouter
+from uuid import UUID
+import app.models.models as models
+from app.database.database import engine, Sessionlocal
+from sqlalchemy.orm import Session
+from app.schemas.schemas import Task, TaskDetails
+from sqlalchemy import desc
+
+
+models.Base.metadata.create_all(bind=engine)
+
+router = APIRouter()
+
+
+def get_db():
+    try:
+        db = Sessionlocal()
+        yield db
+    finally:
+        db.close()
+
+
+# ------------------------------ Get Requests------------------------------
+@router.get("/")
+def get_tasks(db: Session = Depends(get_db)):
+    return db.query(models.Tasks).all()
+
+
+@router.get("/priority-tasks/{priority}")
+def get_tasks(priority: str, db: Session = Depends(get_db)):
+    return db.query(models.Tasks).filter(models.Tasks.priority == priority).all()
+
+
+@router.get("/check-status/{status}")
+def get_tasks(status: bool, db: Session = Depends(get_db)):
+    return db.query(models.Tasks).filter(models.Tasks.status == status).all()
+
+
+@router.get("/sort-by-date/")
+def get_tasks(db: Session = Depends(get_db)):
+    return db.query(models.Tasks).order_by(desc(models.Tasks.start_date)).all()
+
+
+# ------------------------------ Post Requests------------------------------
+
+
+@router.post("/")
+def create_task(task: Task, db: Session = Depends(get_db)):
+    task_model = models.Tasks()
+    task_model.name = task.name
```

Comment:
Do not add fields manually if the model fields and Task schema fields are the same. Read about `*args` & `**kwargs` in Python.